CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contributing
+
+Welcome to Tinted Theming, a community-driven project focused on
+creating and sharing themes. Our mission is to build a diverse
+collection of themes that can be easily adopted and customized. Whether
+you're looking to contribute your existing theme, join as a maintainer,
+help out with design, take part in discussions or report bugs, we're
+excited to have you on board.
+
+## Joining Tinted Theming
+
+Contributors can join the Tinted Theming Github organization in the
+following ways:
+
+1. Join an existing repository as one of the maintainers
+1. Transfer your repository into the Tinted Theming organization and
+   become a maintainer of that repository
+
+## Repository Transfer Process
+
+### Acceptance criteria
+
+Before transferring your repository, please ensure it meets the
+following criteria:
+
+- Follows our [Code of Conduct] and contribution guidelines
+- Aligns with Tinted Theming's thematic and aesthetic standards
+- Includes, at least basic, documentation on theme usage
+
+### Agreement
+
+When a repository is transferred to Tinted Theming, it then belongs to
+the Tinted Theming Github organization and community. The previous owner
+changes from being the owner of the repository to a maintainer of the
+repository. It is not mandatory to be a maintainer of transferring.
+
+Becoming a Tinted Theming maintainer when transfering in a repository is
+optional.
```

Comment:
First point: Yes I think we can add that we reserve the right to remove, but maybe in the `## Maintainer Obligations` section since it relates to maintainers and not specifically people transferring repos. I've put this here for now but can change.

I've added info addressing the other two points over here.